tests/core/test_planetary_return_roundtrip.py

```
@@ -0,0 +1,338 @@
+    with pytest.raises(KerykeionException, match="civil range"):
+        seed("9999-12-31T23:59:59-14:00", backwards=False)  # already year 10000 in UTC
+    with pytest.raises(KerykeionException, match="civil range"):
+        seed("0001-01-01T00:00:00+14:00", backwards=True)  # already year 0 in UTC
+
+
+def test_search_refuses_to_start_outside_the_civil_range(factory):
+    with pytest.raises(KerykeionException, match="civil range"):
+        factory.next_return_from_iso_formatted_time("9999-12-31T23:59:59+00:00", "Solar")
+    # The first second of 1 CE is a valid backward seed; the search itself then
+    # runs out of the civil range and refuses through the library's exception.
+    with pytest.raises(KerykeionException, match="1 CE|civil range|range"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)